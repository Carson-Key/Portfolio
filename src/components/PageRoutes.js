// Packages
import { Routes, Route } from 'react-router-dom'
// Pages
import Projects from '../pages/Projects'
import Experience from '../pages/Experience'
import Education from '../pages/Education'
import Resume from '../pages/Resume'
import About from '../pages/About'

const PageRoutes = () => {
    return (
        <Routes>
            <Route path='/Projects' element={<Projects />} />
            <Route path='/Experience' element={<Experience />} />
            <Route path="/Education" element={<Education />} />
            <Route path="/Resume" element={<Resume />} />
            <Route path="/About" element={<About />} />
            <Route path="/" element={<Projects />} />
        </Routes>
	)
}

export default PageRoutes
