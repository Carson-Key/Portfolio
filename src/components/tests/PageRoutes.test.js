// Packages
import renderer from 'react-test-renderer'
import { BrowserRouter as Router } from 'react-router-dom'
// Pages
import PageRoutes from '../PageRoutes'

describe('PageRoutes Component', () => {
    it('Should Render PageRoutes component', () => {
        const pageroutes = renderer.create(
            <Router>
                <PageRoutes />
            </Router>
        ).toJSON()
        expect(pageroutes).toMatchSnapshot()
    })
})