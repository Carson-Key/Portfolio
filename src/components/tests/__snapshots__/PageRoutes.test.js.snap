// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageRoutes Component Should Render PageRoutes component 1`] = `
<main>
  <p>
    About
  </p>
</main>
`;
